srcpkgs/libcgroup/template

```diff
@@ -23,10 +24,6 @@ esac
 pre_configure() {
 	autoreconf -if
 }
```

Comment:
this shouldn't be necessary

Reply:
I didn't introduce this step and removing it causes the build to fail.

Reply:
If you use a proper release archive, it won't because it includes the configure script:
```
distfiles="https://github.com/libcgroup/libcgroup/releases/download/v${version}/libcgroup-${version}.tar.gz"
```

Reply:
done